### PR TITLE
Added deprecation banners in the overview pages

### PR DESCRIPTION
**Pull request by msewaweru**

### CODE REVIEW

**Comment by billmath** [comment]
Docs Build status updates of commit _[12b8174](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/12b81745389fe7e06a795b5322b2d99883cf26a2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C22%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204221138216303-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-23T11%3A39%3A33Z&sr=b&sp=r&sig=CtAymE9oCuDor91gyz9DzoBzWs6IWTyGmlT1TYDCNic%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[aabf9a1](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/aabf9a11471e9ab8c274de87beeb135e9f23d1a7)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/azureadps-2.0/AzureAD/AzureActiveDirectory.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/index?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) | [Details](#user-content-eac060ef883d0635012c535db595ed74-ae3023e3)
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

<a id="eac060ef883d0635012c535db595ed74-ae3023e3"></a>
### [azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/azureadps-2.0/AzureAD/AzureActiveDirectory.md)
 - **Line 0, Column 0**: **[Warning: file-not-found - [See documentation](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../azureadps-2.0/migration-faq.yml'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C25%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204251637138690-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-26T16%3A38%3A19Z&sr=b&sp=r&sig=Bm85sU2iZlLettAbR6vRJOdNl8%2FUvoxtQXHv5T8h%2B7U%3D).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=main) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=main&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[46d1069](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/46d10696cf5fe0bce954d84e0385ba4b1109261d)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/deprecation-banners/azureadps-2.0/AzureAD/AzureActiveDirectory.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/index?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C25%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204251706227178-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-26T17%3A07%3A48Z&sr=b&sp=r&sig=TRpWGHzeqOzl2NfYspQhy08TO1HyC6DogvOIzE4Fpss%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[d19f72f](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/d19f72fb20439216472d1965db52af491373edf3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/deprecation-banners/azureadps-2.0/AzureAD/AzureActiveDirectory.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/index?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C26%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204260815276577-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-27T08%3A16%3A38Z&sr=b&sp=r&sig=gtTTNt9rhGaeSfrfSS0HqBz3E0Dot%2BpA9vg1BQUHvYM%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
@CelesteDG, please review this.

File: azureadps-2.0/AzureAD/AzureActiveDirectory.md [hunk 1]
```diff
@@ -10,6 +10,11 @@ ms.custom: iamfeature=PowerShell
 ---
 # Azure Active Directory PowerShell for Graph module
 ## Description
+
+>[!IMPORTANT]
+> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```
Comment (CelesteDG) [inline azureadps-2.0/AzureAD/AzureActiveDirectory.md:15]:
```suggestion
> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```
```suggestion
> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../../docs-conceptual/azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```

@mswaweru - Can you please reword the last 3 sentences. There's a lot of passive voice and would be great to see a better description of what the value prop and benefits are for customers using Azure AD PS now and why they need to use MS Graph PS.

File: docs-conceptual/azureadps-2.0-preview/overview.md [hunk 2]
```diff
@@ -10,6 +10,9 @@ ms.custom: posh-docs-conceptual
 ---
 # Azure Active Directory PowerShell for Graph
 
+>[!IMPORTANT]
+> Azure AD PowerShell is planned for deprecation. See the [deprecation update](https://techcommunity.microsoft.com/t5/azure-active-directory-identity/azure-ad-change-management-simplified/ba-p/2967456) for more details on the deprecation plans. We recommend that you start trying [Microsoft Graph PowerShell](/powershell/microsoftgraph/overview). Microsoft Graph PowerShell is the replacement for Azure AD PowerShell and is recommended for interacting with Azure AD. See the [migration FAQ](../azureadps-2.0/migration-faq.yml) for answers to frequent migration queries.
```
Comment (CelesteDG) [inline docs-conceptual/azureadps-2.0-preview/overview.md:14]: Similar comment. Remove the passive voice and provide a more compelling description on "why"

**Review by CelesteDG — CHANGES_REQUESTED** [review]

**Comment by billmath** [comment]
Docs Build status updates of commit _[a774ea0](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/a774ea0483062468c456c72ff5b8e7fb79e75d7f)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/deprecation-banners/azureadps-2.0/AzureAD/AzureActiveDirectory.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/index?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C28%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204281031365612-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-29T10%3A32%3A44Z&sr=b&sp=r&sig=zAjC7sCOW%2FvQ%2BDkEooLJPtzZd7T0mx4xUovKydbY0hY%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by billmath** [comment]
Docs Build status updates of commit _[aeaf56d](https://github.com/msewaweru/azure-docs-powershell-azuread/commits/aeaf56d3e3af539482b9f96612af708d3625b561)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[azureadps-2.0/AzureAD/AzureActiveDirectory.md](https://github.com/msewaweru/azure-docs-powershell-azuread/blob/deprecation-banners/azureadps-2.0/AzureAD/AzureActiveDirectory.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/module/azuread/index?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |
[docs-conceptual/azureadps-1.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-1.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-1.0&branch=pr-en-us-810) (azureadps-1.0) |
[docs-conceptual/azureadps-2.0-preview/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0-preview/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0-preview&branch=pr-en-us-810) (azureadps-2.0-preview) |
[docs-conceptual/azureadps-2.0/overview.md](https://github.com/Azure/azure-docs-powershell-azuread/blob/main/docs-conceptual/azureadps-2.0/overview.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/powershell/azure/active-directory/overview?view=azureadps-2.0&branch=pr-en-us-810) (azureadps-2.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2022%5C4%5C28%5Ced7d034d-a56c-a713-9d51-730d2ba643e8%5CPullRequest%5C202204281720126209-810%5Cworkflow_report.html?sv=2021-04-10&se=2022-05-29T17%3A28%3A52Z&sr=b&sp=r&sig=oJ5UBvCeYhEzEGzaPJuoMJts5zKd3T26n%2FgD3n1a25Y%3D).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/ed7d034d-a56c-a713-9d51-730d2ba643e8?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=main">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by msewaweru** [comment]
@CelesteDG, please review this.

**Pull request merged**